client/src/pages/guide/MoveButtonBox.tsx

```diff
@@ -0,0 +1,74 @@
+import Button from '@/components/Button';
+import { Link } from 'react-router-dom';
+import { PathType } from './type';
+import { useTagSelectContext } from '@/store/useTagSelectContext';
+
+interface MoveButtonProps {
+  path: Exclude<PathType, 'complete'>;
+}
+
+function MoveButtonBox({ path }: MoveButtonProps) {
+  const { selectedAge, selectedGender, selectedKeyword } =
+    useTagSelectContext();
+  return (
+    <>
+      {path === 'age' && (
+        <>
+          {selectedAge !== null ? (
+            <Link to="/model/LX06/guide/gender">
+              <Button size="lg" color="main-blue">
+                다음
+              </Button>
+            </Link>
+          ) : (
+            <Button size="lg" color="sub-blue" disabled={true}>
+              다음
+            </Button>
+          )}
```

Comment:
앞서 말한 것 처럼 disabled css 가상 셀렉터를 사용할 경우, 다음과 같이 간략화 할 수 있을 것 같습니다!

```suggestion
<Button size="lg" color="main-blue" disabled={selectedAge===null}>
  <Link to="/model/LX06/guide/gender">
    다음
  </Link>
</Button>
```

Reply:
수정하였습니다~! 감사합니다